conda_build_config.yaml

```diff
@@ -3,4 +3,9 @@ mpi:
   - openmpi
```

Comment:
Please do not modify files outside of the recipe directory

Reply:
Right, this is already in the recipe directory. Removed modification.